README.md

```diff
@@ -69,7 +69,7 @@ Whether to create a self-signed certificate for serving GitLab over a secure con
     gitlab_ldap_password: "password"
     gitlab_ldap_base: "DC=example,DC=com"
 
-GitLab LDAP configuration; if `gitlab_ldap_enabled` is `true`, the rest of the configuration will tell GitLab how to connect to an LDAP server for centralized authentication.
+GitLab LDAP configuration; if `gitlab_ldap_enabled` is `true`, the rest of the configuration will tell GitLab how to connect to an LDAP server for centralized authentication. The supported LDAP methods are "simple_tls", "start_tls", "plain", "ssl", and "tls". For obvious reasons, plain is the default but should be used with care when authenticating to external LDAP or over untrusted regions. Default gitlab_ldap_uid is "sAMAccountName" for AD, "uid" for most other LDAP.
```

Comment:
Is this documented in GitLab's docs somewhere? I'd much rather link to their documentation for these settings, than have to maintain documentation for GitLab's options in this role.

Reply:
I havent seen it for the CE version, but the EE version does document it here:
https://docs.gitlab.com/ee/administration/auth/ldap.html
https://docs.gitlab.com/ce/administration/auth/ldap.html

I just need a quick refrence for it so either work for me 👍